include/deal.II/trilinos/nox.templates.h

```diff
@@ -1161,12 +1161,21 @@ namespace TrilinosWrappers
                         ExcNOXNoConvergence());
           }
       }
-    // See if NOX returned by triggering an exception. In a sign of generally
-    // poor software design, NOX throws an exception that is not of a class
-    // derived from std::exception, but just a char*. That's a nuisance -- you
-    // just have to know :-(
+    // See if NOX returned by triggering an exception.
+#if DEAL_II_TRILINOS_VERSION_GTE(4, 2, 0)
+    // Starting with Trilinos version 4.2.0 NOX started to throw a
```

Comment:
Do you mean 
```suggestion
#if DEAL_II_TRILINOS_VERSION_GTE(14, 2, 0)
    // Starting with Trilinos version 14.2.0 NOX started to throw a
```
Version 4.2 must be long before I started using Trilinos in 2007, I think the first version I used was 6 or 7.

Reply:
:smile: